routes/index.js

```diff
@@ -287,28 +289,32 @@ router.get('/search', function(req, res, next) {
     function findSongsLoggedIn(done) {
         var query = new RegExp('.*' + searchString + '.*')
         Song.find({
-                    $or: [{
-                        title: {
-                            $regex: query,
-                            $options: 'si' //s option to allow dot character to match all characters including new line
-                        }
-                    }, {
-                        author: {
-                            $regex: query,
-                            $options: 'si'
-                        }
-                    }, {
-                        lyric: {
-                            $regex: query,
-                            $options: 'si'
-                        }
-                    }, {
-                        copyright: 'private',
-                        contributor: req.user.username
+                    $and: [{
+                        $or: [{
+                            title: {
+                                $regex: query,
+                                $options: 'si' //s option to allow dot character to match all characters including new line
+                            }
+                        }, {
+                            author: {
+                                $regex: query,
+                                $options: 'si'
+                            }
+                        }, {
+                            lyric: {
+                                $regex: query,
+                                $options: 'si'
+                            }
+                        }]
                     }, {
-                        copyright: {
-                            $ne: 'private'
-                        }
+                        $or: [{
+                            copyright: 'private',
```

Comment:
so you want the copyright lists which are cc0, public, and private stored in array and everything just refers to it?